api/src/main/java/com/google/sps/SendNotifications.java

```diff
@@ -79,7 +79,7 @@ public Email getNextRecipient(HttpServletRequest request, HttpServletResponse re
     try {
       DocumentSnapshot gameDocSnap = game.get().get();
       int currentPlayerIndex = (int) ((long) gameDocSnap.get("currentPlayerIndex"));
-      String playerName = ((List<String>) gameDocSnap.get("players")).get(currentPlayerIndex);
+      String playerName = ((List<String>) gameDocSnap.get("players")).get(currentPlayerIndex+1);
```

Comment:
Nit: I would put spaces around the plus sign like this `currentPlayerIndex + 1` just for better style and ease of reading.

Reply:
Also I'm wondering if this is a race condition if it worked before but needed to be fixed now lol